pillar/params.sls

```diff
@@ -67,6 +67,7 @@ kube_log_level:   '2'
 
 # install the addons (ie, DNS)
 addons:
+  psp:    'true'
```

Comment:
Is the PodSecurityPolicy really a addon? Is it optional? Are we going to expose this flag to the user?

Reply:
I'd like it to be optional yes, although I don't think we want to expose that in Velum - I just want is as a "advanced" option for anyone who happens to need it disabled for whatever reason.